.github/workflows/camel_version.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-        camel-version: [ "4.0.0",  "4.0.0.redhat-00036" ]
+        camel-version: [ "4.0.0",  "4.0.0.redhat-00036", "4.4.0.redhat-00019"]
```

Comment:
initially we wanted to be aligned for testing with redhat build + appropriate upstream version... regarding that.. wdyt to have matrix only with `4.4.0` and `4.4.0.redhat-00019` ?